crates/router/src/core/admin.rs

```diff
@@ -146,6 +144,19 @@ pub async fn create_merchant_account(
         })
         .transpose()?;
 
+    let organization_id = if let Some(organization_id) = req.organization_id.as_ref() {
+        organization_id.to_string()
```

Comment:
should we check if this a valid org_id ?

Reply:
Yes we should